### ISSUE TITLE
Game.update() failure.

### ISSUE THREAD

**Issue by Pixelsnarfer** [itdelatrisu/opsu#189]
**Version:** 0.12.1
**Build date:** 2015-10-02 03:05
**OS:** Windows 8.1 (x86)
**JRE:** 1.8.0_73
**Stack trace:**

```
org.newdawn.slick.SlickException: Game.update() failure.
    at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:708)
    at itdelatrisu.opsu.Container.gameLoop(Container.java:98)
    at itdelatrisu.opsu.Container.start(Container.java:72)
    at itdelatrisu.opsu.Opsu.main(Opsu.java:205)
Caused by: java.lang.NullPointerException
    at itdelatrisu.opsu.db.ScoreDB.getMapSetScores(ScoreDB.java:358)
    at itdelatrisu.opsu.states.SongMenu.enter(SongMenu.java:1147)
    at org.newdawn.slick.state.StateBasedGame.update(StateBasedGame.java:248)
    at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:702)
    ... 3 more
```

**Comment by mad-s** [comment]
You probably have a beatmap file with an invalid formatted line `Version:` with nothing following (It's probably one you played the last time you started opsu!). You can manually edit the database to remove the score, and remove the offending beatmap.

Note: this is fixed easily, just initialize `version` to `null` instead of the empty string in `ScoreDB.java:349` since no string equals `null`.

**Comment by itdelatrisu** [comment]
Oops, forgot about this.  Was fixed in #219 as stated in the post above.
